examples/Finding_Local_Peaks_in_Solar_Data.py

```diff
@@ -0,0 +1,82 @@
+"""
+=================================
+Finding Local Peaks in Solar Data
+=================================
+
+Detecting radiation peaks in the solar surface is often crucial in the study of solar flares.
```

Comment:
"Detecting intensity peaks in solar images can be useful, for example as a simple flare identification mechanism."